lib/src/DateRangePicker/DateRangePickerViewDesktop.tsx

```diff
@@ -0,0 +1,176 @@
+import * as React from 'react';
+import { DateRange } from './RangeTypes';
+import { DateRangeDay } from './DateRangePickerDay';
+import { useUtils } from '../_shared/hooks/useUtils';
+import { makeStyles } from '@material-ui/core/styles';
+import { MaterialUiPickersDate } from '../typings/date';
+import { calculateRangePreview } from './date-range-manager';
+import { Calendar, CalendarProps } from '../views/Calendar/Calendar';
+import { isWithinRange, isStartOfRange, isEndOfRange } from '../_helpers/date-utils';
+import { ArrowSwitcher, ExportedArrowSwitcherProps } from '../_shared/ArrowSwitcher';
+import {
+  usePreviousMonthDisabled,
+  useNextMonthDisabled,
+} from '../_shared/hooks/date-helpers-hooks';
+
+export interface ExportedDesktopDateRangeCalendarProps {
+  /**
+   * How many calendars render on **desktop** DateRangePicker
+   * @default 2
+   */
+  calendars?: 1 | 2 | 3;
+}
+
+interface DesktopDateRangeCalendarProps
+  extends ExportedDesktopDateRangeCalendarProps,
+    CalendarProps,
+    ExportedArrowSwitcherProps {
+  date: DateRange;
+  changeMonth: (date: MaterialUiPickersDate) => void;
+  currentlySelectingRangeEnd: 'start' | 'end';
+}
+
+export const useStyles = makeStyles(
+  theme => ({
+    dateRangeContainer: {
+      display: 'flex',
+      flexDirection: 'row',
+    },
+    rangeCalendarContainer: {
+      '&:not(:last-child)': {
+        borderRight: `2px solid ${theme.palette.divider}`,
+      },
+    },
+    calendar: {
+      minWidth: 312,
+      minHeight: 288,
+    },
+    arrowSwitcher: {
+      padding: '16px 16px 8px 16px',
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'space-between',
+    },
+  }),
+  { name: 'MuiPickersDesktopDateRangeCalendar' }
+);
+
+export const DateRangePickerViewDesktop: React.FC<DesktopDateRangeCalendarProps> = ({
+  date,
+  calendars = 2,
+  changeMonth,
+  leftArrowButtonProps,
+  leftArrowButtonText,
+  leftArrowIcon,
+  rightArrowButtonProps,
+  rightArrowButtonText,
+  rightArrowIcon,
+  onChange,
+  disableFuture,
+  disablePast,
+  minDate,
+  maxDate,
+  currentlySelectingRangeEnd,
+  ...CalendarProps
```

Comment:
```suggestion
  ...other
```

Reply:
this spreading rename makes code more readable and strict, because we know that there should be only `Calendar` props

Reply:
In this case, I propose we introduce a new prop `CalendarProps`. So we can keep other spreading on the root element.